test/unit/object/test_upload_session.py

```diff
@@ -0,0 +1,116 @@
+# coding: utf-8
+
+from __future__ import unicode_literals, absolute_import
+
+import base64
+import hashlib
+from io import BytesIO
+import json
+import pytest
+
+from boxsdk.config import API
+from boxsdk.object.file import File
+from boxsdk.object.upload_session import UploadSession
+
+
+@pytest.fixture()
+def test_upload_session(mock_box_session):
+    upload_session_response_object = {
+        'part_size': 8,
+        'total_parts': 10,
+    }
+    return UploadSession(mock_box_session, '11493C07ED3EABB6E59874D3A1EF3581', upload_session_response_object)
+
+
+def test_get_parts(test_upload_session, mock_box_session):
+    expected_url = '{0}/files/upload_sessions/{1}/parts'.format(API.UPLOAD_URL, test_upload_session.object_id)
+
+    parts = mock_box_session.get.return_value.json.return_value = {
+        'entries': [
+            {
+                'part': {
```

Comment:
According to the API reference, this isn't what the response looks like.  Please align with the actual response format and ensure that this still works correctly